components/amorphic/lib/routes/processFile.js

```diff
@@ -26,37 +26,47 @@ function processFile(req, resp, next, downloads) {
 
     let form = new formidable.IncomingForm();
     form.uploadDir = downloads;
-
     form.parse(req, function ee(err, _fields, files) {
-        if (err) {
-            logMessage(err);
-        }
-
-        resp.writeHead(200, {'content-type': 'text/html'});
+        try {
+            if (err) {
+                logMessage(err);
+            }
 
-        let file = files.file.path;
-        logMessage(file);
+            let file = files.file.path;
+            resp.writeHead(200, {'content-type': 'text/html'});
+            logMessage(file);
+        
+            setTimeout(function yz() {
+                fs.unlink(file, function zy(err) {
+                    if (err) {
+                        logMessage(err);
+                    }
+                    else {
+                        logMessage(file + ' deleted');
+                    }
+                });
+            }, 60000);
 
-        setTimeout(function yz() {
-            fs.unlink(file, function zy(err) {
-                if (err) {
-                    logMessage(err);
-                }
-                else {
-                    logMessage(file + ' deleted');
-                }
-            });
-        }, 60000);
+            let fileName = files.file.name;
```

Comment:
referring back to DM - let's add some good null checking here so we can be more targeted in our error handling / messaging if something goes wrong at this line

Reply:
updated logic to only handle null check case